lightly/api/api_workflow_predictions.py

```diff
@@ -0,0 +1,155 @@
+from concurrent.futures import ThreadPoolExecutor
+from typing import Dict, List, Union, Optional
+
+import tqdm
+
+from lightly.api.prediction_singletons import PredictionSingletonClassificationRepr
+from lightly.api.utils import retry
+from lightly.openapi_generated.swagger_client import (
+    PredictionTaskSchema,
+    SamplePartialMode,
+    PredictionSingletonClassification,
+    PredictionSingletonObjectDetection,
+    PredictionSingletonInstanceSegmentation,
+    PredictionSingletonKeypointDetection,
+)
+
+
+class _PredictionsMixin:
+    def create_or_update_prediction_task_schema(
+        self,
+        schema: PredictionTaskSchema,
+        prediction_uuid_timestamp: int,
+    ) -> None:
+        """Creates or updates the prediction task schema
+
+        Args:
+            schema:
+                The prediction task schema.
+            prediction_uuid_timestamp:
+                The uuid timestamp in nanoseconds since 1970. Get it via 'int(time.time())'.
```

Comment:
its important to know that this is the unique identifier (that is also a timestamp) and not just a random timestamp. If its nanoseconds, milliseconds or seconds is very irrelevant. The most important aspect is that this the `key` for everything happening around the schemas and the singletons.

Reply:
I changed the docstring accordingly.